Source/CardView.h

```diff
@@ -25,5 +25,7 @@
 @optional
```

Comment:
What's up with your API? Do you need all those insert/remove methods? Don't you just need collapse ones?